legate/core/__init__.py

```diff
@@ -17,9 +17,33 @@
 from ..rc import check_legion
 from ..util.args import parse_library_command_args
 
-check_legion()
```

Comment:
We can probably remove `legate/rc.py` altogether now.

Reply:
removed.